modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -29,7 +29,8 @@ endif::[]
 [id="installation-adding-registry-pull-secret_{context}"]
 = Configuring credentials that allow images to be mirrored
 
-Create a container image registry credentials file that enables you to mirror images from Red{nbsp}Hat to your mirror.
+[role="_abstract"]
+Create a container image registry credentials file that enables you to mirror images from Red{nbsp}Hat to your mirror. . Complete the following steps on the installation host.
```

Comment:
```suggestion
Create a container image registry credentials file that enables you to mirror images from Red{nbsp}Hat to your mirror. Complete the following steps on the installation host.
```

Reply:
"Enables you" might be classed as anthrophomorphism. How about:

> Create a container image registry credentials file, so that you can mirror images from Red{nbsp}Hat to your mirror registry. Ensure that you completed the procedure steps on the installation host.